tests/pytest_tests/system_tests/test_core/test_keras_full.py

```diff
@@ -149,7 +157,13 @@ def test_keras_image_multiclass(dummy_model, dummy_data, relay_server, wandb_ini
             epochs=2,
             batch_size=36,
             validation_data=dummy_data,
-            callbacks=[WandbCallback(data_type="image", predictions=10)],
+            callbacks=[
+                WandbCallback(
+                    input_type="image",
+                    predictions=10,
+                    validation_data=dummy_data,
```

Comment:
`validation_data` should be picked up by our monkey patching. We should verify that the new EvalCallback still works, too.
cc @morganmcg1